python/mach_bootstrap.py

```diff
@@ -160,7 +160,22 @@ def _activate_virtualenv(topdir):
         open(marker_path, 'w').close()
 
 
+def _ensure_case_insensitive_if_windows():
+    # The folder is called 'python'. By deliberately checking for it with the wrong case, we determine if the file
```

Comment:
Yes, assuming that the reader has a basic knowledge of the folder structure in the project. I don't know, I feel like it's better to be "too explicit" in cases like this. It feels more friendly to the casual reader that way. 😇